.eslintrc.js

```diff
@@ -14,6 +14,9 @@ module.exports = {
   env: {
     browser: true
   },
+  globals: {
+    basicContext: false
```

Comment:
🤔 what is `basicContext`?

Reply:
It is the new contextMenu plugin we are using.